parlai/core/torch_classifier_agent.py

```diff
@@ -287,7 +287,7 @@ def eval_step(self, batch):
 
         if batch.labels is None or self.opt['ignore_labels']:
             # interactive mode
-            if self.opt.get('interactive_mode', False):
+            if self.opt.get('print_scores', False):
```

Comment:
should print scores be set automatically true in interactive?

Reply:
No I don't think so. We don't automatically want to return the scores.  Most often we just want the class prediction.